src/components/inputs/Toggle/Toggle.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * Toggle component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {boolean} [props.disabled]  Flag indicating toggle disabled
+ * @param {string} props.name  Toggle name
+ * @param {Function} [props.onChange]  Action to perform on toggle change
+ * @param {boolean} [props.required]  Flag indicating toggle required
+ * @param {boolean} props.value  Toggle value
```

Comment:
` value` was meant to be used as the label displayed next to the toggle. Looking at it, it doesn't make sense to have `name` and `value`. We can use `name` as the label to display next to the toggle and get rid of this prop altogether.

Reply:
My understanding was that this value prop indicates whether the input is checked or not receiving the value true or false. If this prop does not have this role then depending on what I know this toggle is true or false?